embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/network/logging/EmbraceNetworkLoggingService.kt

```diff
@@ -68,8 +74,6 @@
             logger.logError(msg, IllegalStateException(msg), true)
         }
 
-        // clear calls per domain and session network calls lists before be used by the next session
```

Comment:
Lol is one line is probably the most important change in the PR. We were resetting this every time a session payload is built, which effectively nerfs the limit.

Reply:
Periodic session caching strikes again 💥